Bottom: 514ea88f8631add7f29fd4864109d06c586ef744
Top:    8989932d0d63104560d741afe9431116f1d794a8
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-05 23:41:00 +0100

Refresh of add-a-hid-api-call-notify_save

---

diff --git a/src/file.c b/src/file.c
index 7c1ebc0..4297244 100644
--- a/src/file.c
+++ b/src/file.c
@@ -370,7 +370,7 @@ SavePCB (char *Filename)
     }
 
   if (gui->notify_save_pcb != NULL)
-    gui->notify_save_pcb (Filename, true);
+    gui->notify_save_pcb (PCB->Filename, true);
 
   return (retcode);
 }
